Libraries/plist/CMakeLists.txt

```diff
@@ -66,7 +66,11 @@ if ("${CMAKE_SYSTEM_NAME}" MATCHES "Windows")
   endif ()
 else ()
   find_package(LibXml2 REQUIRED)
-  target_include_directories(plist PRIVATE "${LIBXML2_INCLUDE_DIR}")
+  if ("${CMAKE_SYSTEM_NAME}" MATCHES "OpenBSD")
+    target_include_directories(plist PRIVATE "${LIBXML2_INCLUDE_DIRS}")
```

Comment:
I don't understand this. Does cmake behave differently on OpenBSD?

Reply:
ref:http://openbsd-archive.7691.n7.nabble.com/cmake-woes-wrt-libxml2-include-dirs-td321941.html

Reply:
I see. Thanks for the link.